src/ros_interop/map_image_manager.py

```diff
@@ -0,0 +1,34 @@
+import rospy
+from ros_interop.msg import *
+from ros_interop.srv import ImageResponse
+
+class MapImageManager():
+    def __init__(self,interop_client) -> None:
```

Comment:
you should run [black](https://github.com/psf/black) over your code to format it nicely